CMakeLists.txt

```diff
@@ -18,6 +13,18 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=0")
 # We require a C++11 compliant compiler
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# supress warning of ray_shared
+cmake_policy(SET CMP0068 OLD)
```

Comment:
Add a comment to explain what the warning is?

Reply:
I have no idea where `ray_shared` target is setup.

anybody knows how to remove the warning of ray_shared about MACOS RPATH?